src/api/status/package.json

```diff
@@ -15,7 +17,15 @@
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
     "@senecacdot/satellite": "^1.x",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "gulp": "^4.0.2",
```

Comment:
These are all `devDependencies`, no?

Reply:
https://github.com/Seneca-CDOT/telescope/pull/2522#discussion_r756479096

Reply:
@humphd e2e doesn't install dev dependencies so it crashes the tests. 

https://github.com/Seneca-CDOT/telescope/pull/2522#discussion_r756479096

Reply:
Another way is manually run the sass compiler
```
- npm run compile:scss
+ npm run sass .\scss\material-dashboard.scss:.\public\assets\css\material-dashboard.css
```

Reply:
It's not a huge deal if we have to do it this way.